src/CopyrighFixer/CopyrighFixer/configparser.cpp

```diff
@@ -0,0 +1,50 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "configparser.h"
+#include <quasarapp.h>
+#include <iostream>
+
+
+namespace CopyrighFixer {
+ConfigParser::ConfigParser() {
+
+}
+
+const Config ConfigParser::parseOptions(const Config &conf) const {
+
+    Config configOption = conf;
+
+    if (QuasarAppUtils::Params::isEndable("sourceDir")) {
+        configOption.setSourceDir(QuasarAppUtils::Params::getArg("sourceDir"));
+    }
+    else {
+        std::cout << "Warning: Not option sourceDir" << std::endl;
+    }
+
+    if (QuasarAppUtils::Params::isEndable("sign")) {
+        Signature *signature = new Signature;
+        configOption.setSingValue(*signature);
```

Comment:
This is memory leak. Because you create signature object on heap (line 30), invoke a copy constructor (line 31) but not remove it.
  
In this case you need to create the  signature object on stack.